tcms/core/ajax.py

```diff
@@ -240,6 +198,57 @@ def remove(self):
     return render(request, template_name, context_data)
 
 
+class _TagObjects(object):
+    """ Used for getting the chosen object(TestPlan, TestCase or TestRun) from the database """
+
+    def __init__(self, request):
+        """
+        :param request: An HTTP GET request, containing the private key
```

Comment:
definitely not a private key but rather a primary key of the object we are looking for.